php/one-fpm/config.yaml

```diff
@@ -0,0 +1,4 @@
+framework:
+  github: lizhichao/one
+  version: 1.8
+  
```

Comment:
could you add
~~~yaml
before_command:
  - mkdir -p App/RunCache/cache
  - chmod 777 App/RunCache/cache
~~~
in order to create the **cache** directory